projects/core/src/site-context/services/site-context-params.service.ts

```diff
@@ -62,4 +64,25 @@ export class SiteContextParamsService {
       service.setActive(value);
     }
   }
+
+  /**
+   * Get active values for all provided context parameters
+   *
+   * @param params Context parameters
+   *
+   * @returns Observable emitting array of all passed active context values
+   */
+  getValues(params: string[]): Observable<Array<string>> {
+    if (params.length === 0) {
+      return of([]);
+    }
+
+    return combineLatest(
```

Comment:
why did you use combineLatest with one parameter?

Reply:
Params is an array.